webroot/src/store/user/user.actions.ts

```diff
@@ -32,10 +32,15 @@ export default {
         commit(MutationTypes.LOGIN_FAILURE, error);
     },
     // LOGOUT
-    logoutRequest: ({ commit, dispatch }) => {
+    logoutRequest: ({ commit, dispatch }, authType) => {
         dispatch('clearSessionStores');
         dispatch('startLoading', null, { root: true });
-        dispatch('clearAuthTokens');
+        let tokenType = AuthTypes.STAFF;
+
+        if (authType === AuthTypes.LICENSEE) {
+            tokenType = authType;
+        }
```

Comment:
Mostly, yes. By doing it this way rather than just passing that value through we are:
1) Defaulting to logout staff if no value is passed in. The logoutRequest call gets made with no inputs in multiple places throughout the project atm. This could be also be achieved by setting a default value to the input
2) Defaulting to staff if the input AuthType doesn't match the options. If null, '', or any arbitrary value is passed in as authtype it'll likely throw an error. With this implementation it will simply default to logging staff out.

Im open to arguments that 2 is not a virtue. I'm personally ambivalent.